article/app/controllers/ArticleController.scala

```diff
@@ -144,4 +146,11 @@ class ArticleController(
     }
   }
 
+  def servePressedPage(path: String)(implicit request: RequestHeader): Future[Result] = {
+    val cacheable = WithoutRevalidationResult(
+      Ok.withHeaders("X-Accel-Redirect" -> s"/s3-archive/www.theguardian.com/$path"),
```

Comment:
We serve a redirect here because that was the advice we got when implementing equivalent logic for the interactive migration (if I understood correctly, serving the redirect is quicker?).

Note that this redirect is for the prod s3 bucket (https://github.com/guardian/platform/blob/main/router/files/router.conf#L671). We did wonder if we should configure the redirect to work for CODE (i.e. if stage is CODE then point to equivalent CODE s3 bucket). However, there is likely an upcoming improvement we'd like to make where:
- try to get doc from s3
- if success --> serve
- if failure --> render via dcr (even if it's broken, but that's better than serving a 500?)

Reply:
If this is the logic for interactives I'd feel comfortable doing the same for articles, but the suggestion for in case is does fail sounds worth looking into!